locallibrary/catalog/forms.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from django import forms
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+class RenewBookForm(forms.Form):
+    renewal_date = forms.DateField(help_text="Enter a date between now and 4 weeks (default 3).")
+
+    def clean_renewal_date(self):
+        data = self.cleaned_data['renewal_date']
```

Comment:
lấy dữ liệu dùng hàm cleaned_data
gán biến data để sau khi validate xong. thì trả lại dữ liệu data (cái này để bảo toàn về mặt dữ liệu k bị thay đổi khi validate)